postgres-consistency/scripts/EpochSlotToBlkHash.js

```diff
@@ -0,0 +1,82 @@
+var fetch = require("node-fetch");
+
+const serverRoute = 'https://explorer.iohkdev.io/api';
+
+const slotsPerEpoch = 21600;
+
+const randomNumberWithMax = function (topNumber) {
```

Comment:
I think is a good idea to add a `TODO` saying that the random method used could be improved.